templates/pioneer-meta/pioneer-enrollment/pioneer-enrollment.1.schema.json

```diff
@@ -0,0 +1,8 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "title": "pioneer-enrollment",
+  "$comment": "Bug 1663857 - Add pioneer-enrollment pings for explict consent logging.",
+  "properties": {},
+  "additionalProperties": false
+}
```

Comment:
Given that this ping is duplicated in many namespaces, can we factor this whole structure out to `templates/include/pioneer` for consistency?